### ISSUE TITLE
Missing location on /tickets

### ISSUE THREAD

**Issue by racke** [interchange/Perl-Dancer-Conference#54]
One user replied to my mailing post advertising tickets and said that it isn't clear from

https://www.perl.dance/tickets

where the event takes place. 

I suggest to add "Vienna/Austria" to the small logo in the upper left as well.

Thoughts?

**Comment by sbatschelet** [comment]
Makes sense who has the source on this media @yure I assume?  Care to share your work kind sir or at least what is the font?

**Comment by yure** [comment]
this page is intended for people to buy. less distractions the better.

ok reviews build trust, but i think if they clicked tickets they already 
know the location

**Comment by sbatschelet** [comment]
@yure :gun: font sir what is the font.

**Comment by sbatschelet** [comment]
I agree with you @yure but if we are linking out the ticket route why not give just a tiny reminder :)

**Comment by racke** [comment]
I was talking about the small logo, can't see why this is a distraction.

**Comment by yure** [comment]
Where it says "Conference Only" add one line below saying "Hotel Schani, Wien", where it says conference + training add "Hotel Schani + Amerikahaus, Wien"
